moveit_ros/moveit_servo/README.md

```diff
@@ -1,3 +1,3 @@
 # Moveit Servo
 
-See the [Realtime Arm Servoing Tutorial](http://moveit2_tutorials.picknik.ai/doc/realtime_servo/realtime_servo_tutorial.html) for installation instructions, quick-start guide, an overview about `moveit_servo`, and to learn how to set it up on your robot.
+See the [Realtime Arm Servoing Tutorial](https://moveit.picknik.ai/foxy/doc/realtime_servo/realtime_servo_tutorial.html) for installation instructions, quick-start guide, an overview about `moveit_servo`, and to learn how to set it up on your robot.
```

Comment:
Do we have a way to link to the most recent version of the tutorial such that this doesn't periodically go stale?

Reply:
No, there isn't a way to do that afaik, right now there are also no other versions of the tutorials so this is fine.